src/index.js

```diff
@@ -173,11 +177,16 @@ function Player(opts) {
 util.inherits(Player, events.EventEmitter);
 
 Player.prototype.init = function(opts) {
+  let that = this;
   this.serviceName = `org.mpris.MediaPlayer2.${this.name}`;
   dbus.validators.assertBusNameValid(this.serviceName);
 
   this._bus = dbus.sessionBus();
 
+  this._bus.connection.on('error', (err) => {
+    that.emit('error', err);
```

Comment:
No need for this, arrow functions don't shallow `this`

Reply:
Or should I say, no need for _that_

Reply:
:+1: